packages/atlas-meta-server/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@joystream/atlas-meta-server",
-  "version": "1.2.1",
+  "version": "1.3.0",
```

Comment:
Not sure if you want to keep this in sync with main app's version

Reply:
Own versioning system seems a better option, as it allows to track changes separately